### ISSUE TITLE
Get banned from instagram !

### ISSUE THREAD

**Issue by harunemre** [ramboxapp/community-edition#2931]
### Steps to reproduce

1. Set up instragram direct
2. Login successfully
3. A few hours later my account get banned

### ENVIRONMENT

> Rambox: v0.7.7

> OS: windows 10

> Arch: x64

**Comment by VirtualDOMinic** [comment]
I also had this issue with my instagram account. I have two accounts, neither of which I use any other third party software on, neither of which has ever been suspended, and neither of which does any spammy/dodgy activity. 

The one I've used Rambox on suddenly got banned earlier this week, shortly after logging in on Rambox mac (although I use it on both mac and PC). The one that I've never used via Rambox is alive and well, and is even linked to the other account.

This is quite a big issue, e.g. for me I'm now worried I will lose easy contact to friends I've made through my travels in the last few years. Friends I don't have on other messaging services.

**Comment by harunemre** [comment]
@VirtualDOMinic i agree with you mate. Its so sad and i had lost access to instagram since i've used rambox about 4 days ago.

Currently im in a "help us confirm you own this account" loop showed as image blow. I tried to reset my password, contact with help service of instagram with no chance.

![image](https://user-images.githubusercontent.com/27810645/102897245-bf3ad400-4478-11eb-854b-9ae32bcc779b.png)

**Comment by VirtualDOMinic** [comment]
More cases of people having this issue recently: https://www.reddit.com/r/Instagram/comments/jirjb2/warning_rambox_app_instagram_disabled_account/

@harunemre if you haven't tried already, be sure to at least appeal via the "learn more" link that appears in the app when you try to log in. You should be taken to a page where you provide some basic details, and then you should get an email asking for details and a photo of yourself holding a code. That's the process that I'm currently going through, but I'm about to reach 48 hours with no response. Might try again shortly

**Comment by VirtualDOMinic** [comment]
Just to add, this is the URL I get taken to to appeal the ban: https://www.facebook.com/help/instagram/521372114683554

My hypothesis (no evidence at this stage) is that some software for getting more followers/likes has forked a lot of the rambox code, or that a service for spamming likes on people's behalf is using rambox to log in to dozens of accounts simultaneously. Either of these options might have resulted in Instagram's spam algorithm treating rambox's signature as suspicious...

**Comment by harunemre** [comment]
@VirtualDOMinic thanks mate i try your suggested way but it showed that "your account is already active"

**Comment by fvulich** [comment]
We're investigating this issue. We'll come back with some feedback.

**Comment by VirtualDOMinic** [comment]
Thanks Vulich! A tiny bit of additional info: I also was asked to verify my phone number and that I'm not a robot a couple of times (via image captchas) shortly before the "ban", similar to what harun shows in their screenshot

**Comment by dandv** [comment]
I almost got banned from Messenger today. Entered the 2FA code, then Messenger said my account was locked (!) and I had to complete some step (forgot what the step was). Fortunately after I completed the step, I have access to Messenger.

**Comment by harunemre** [comment]
Update: I had no chance to unlock my banned instagram account. So i had to create a new one. My locked account seems like get lost in the dusty shelves of history 😞

**Comment by rauldipeas** [comment]
I lost my Instagram account too, after start using it on Rambox, for no reason.

**Comment by civilalloy** [comment]
Got banned in October using Rambox :( i've been trying to get my account back for months without any luck.

**Comment by marcosimprota** [comment]
ive been trying to get my instagram account back for months and months!.!"Help Us Confirm You Own This Account" loop keep happening over over

**Comment by VirtualDOMinic** [comment]
I managed to get my account reinstated but only because my friend got her ex who works at facebook to make an internal ticket for me... 😆 . Unfortunately they didn't provide any useful info about why the ban happened, so I can't help shed light on what could cause this if Rambox is the cause.

**Comment by civilalloy** [comment]
> I managed to get my account reinstated but only because my friend got her ex who works at facebook to make an internal ticket for me... 😆 . Unfortunately they didn't provide any useful info about why the ban happened, so I can't help shed light on what could cause this if Rambox is the cause.

Idk if this is against the rules here but I'll shoot my shot. My Instagram account was very important to me. Can I please pay you some money or your friend's ex at fb to do this for my account. I'm desperate at this point

**Comment by harunemre** [comment]
Update: After 7 months later worlds worst company facebook decided to send me a recover email by suprise 😨 So i got my old account back now but dont know what to do cause i have been using my new account since old one was suspended.
